buildroot-external/configs/generic_x86_64_defconfig

```diff
@@ -126,6 +126,8 @@ BR2_PACKAGE_IPTABLES_NFTABLES=y
 BR2_PACKAGE_IPTABLES_NFTABLES_DEFAULT=y
 BR2_PACKAGE_NETWORK_MANAGER=y
 BR2_PACKAGE_NETWORK_MANAGER_CLI=y
+BR2_PACKAGE_DNSMASQ=y
+BR2_PACKAGE_DNSMASQ_TFTP=n
```

Comment:
This is not correct, as you can see above, disabling should be done through the `# ... is not set` stanza.